src/geometries/primitives/parametrizedcurve.jl

```diff
@@ -0,0 +1,41 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    ParametrizedCurve(a, b, func)
+
+A parametrized curve is a curve defined by a function `func` that maps a parameter `t` to a `Point` in space.
+The parameter `t` is defined in the interval `[a, b]`.
+
+## Examples
+
+```julia
+ParametrizedCurve(0.0, 1.0, t -> Point(cospi(2t), sinpi(2t)))
+```
+"""
+struct ParametrizedCurve{M<:Meshes.Manifold,C<:Meshes.CRS,T<:Real,F<:Function} <: Primitive{M,C}
+  a::T
+  b::T
+  func::F
+
+  function ParametrizedCurve(a::T, b::T, func) where {T<:Real}
+    p = func(a)
+    M = manifold(p)
+    C = crs(p)
+    new{M,C,T,typeof(func)}(a, b, func)
+  end
+end
+
+paramdim(::ParametrizedCurve) = 1
+startparameter(curve::ParametrizedCurve) = curve.a
+startpoint(curve::ParametrizedCurve) = curve(curve.a)
```

Comment:
```suggestion
minimum(curve::ParametrizedCurve) = curve(curve.a)
```

We use `minimum`, `maximum` and `extrema` in other primitives to retrieve the extrema. Maybe we should use the same names here.

Reply:
Done in e61f8fc378be948a9c60955013eef815e82a2cf3.